orchestration/dagster_orchestration/hca_manage/main.py

```diff
@@ -75,6 +75,11 @@ def run(arguments: Optional[list[str]] = None) -> None:
     dataset_delete_args.add_argument("-n", "--dataset_name", help="Name of dataset to delete.")
     dataset_delete_args.add_argument("-i", "--dataset_id", help="ID of dataset to delete.")
 
+    parser_soft_delete = subparsers.add_parser("soft_delete", help="Soft delete rows")
+    parser_soft_delete.add_argument("-p", "--path", help="Path to csv containing row IDs to soft delete")
+    parser_soft_delete.add_argument("-t", "--target_table", help="Table containing the rows slated for soft deletion")
```

Comment:
oop, missed my favorite nit: we should be using `kebab-case` for command line argument flags

Reply:
Yeah...I noticed that too. I think this should remain snake to retain consistency with the other args this tool consumes, we can kebabify as part of DSPDC-1688